[2025-04-02T14:50:28.034+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-02T14:50:28.108+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LocalIngestionDag.download_dataset scheduled__2025-03-02T06:00:00+00:00 [queued]>
[2025-04-02T14:50:28.161+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LocalIngestionDag.download_dataset scheduled__2025-03-02T06:00:00+00:00 [queued]>
[2025-04-02T14:50:28.163+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-02T14:50:28.225+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): download_dataset> on 2025-03-02 06:00:00+00:00
[2025-04-02T14:50:28.249+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=893) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-02T14:50:28.255+0000] {standard_task_runner.py:72} INFO - Started process 898 to run task
[2025-04-02T14:50:28.295+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'download_dataset', 'scheduled__2025-03-02T06:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmplhx2yb7w']
[2025-04-02T14:50:28.322+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask download_dataset
[2025-04-02T14:50:28.690+0000] {task_command.py:467} INFO - Running <TaskInstance: LocalIngestionDag.download_dataset scheduled__2025-03-02T06:00:00+00:00 [running]> on host 6baa03efc864
[2025-04-02T14:50:28.917+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='LocalIngestionDag' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-02T06:00:00+00:00'
[2025-04-02T14:50:28.922+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-02T14:50:28.956+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-02T14:50:28.958+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello, world"']
[2025-04-02T14:50:28.979+0000] {subprocess.py:99} INFO - Output:
[2025-04-02T14:50:28.981+0000] {subprocess.py:106} INFO - Hello, world
[2025-04-02T14:50:28.983+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-02T14:50:29.026+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-02T14:50:29.028+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag, task_id=download_dataset, run_id=scheduled__2025-03-02T06:00:00+00:00, execution_date=20250302T060000, start_date=20250402T145028, end_date=20250402T145029
[2025-04-02T14:50:29.078+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-02T14:50:29.128+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-02T14:50:29.132+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
